### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by KeatonSushi**
added temporary images to the about us page and added content from old site.

### CODE REVIEW

File: aboutUsNew.html [hunk 1]
```diff
@@ -1,86 +1,176 @@
...
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="#">Donate</a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="#">About Us</a>
-              </li>
-            </ul>
-          </div>
+      <div class="container-fluid">
```
Comment (LitterPic) [inline aboutUsNew.html:35]: I have a navbar.html and an include-navbar.js defined.  Are you defining a new navbar?

**Comment by KeatonSushi** [comment]
I did a copy and paste from the index.html file you had.

**Comment by LitterPic** [comment]
Sorry I’m not understanding.  I have a navbar set up that is shared across all screens, it looks like you’re implementing a new nav bar in a page called AboutUsNew.html

**Comment by KeatonSushi** [comment]
Oh okay. How do I copy and paste your original navbar?

**Comment by KeatonSushi** [comment]
Or where should I copy it from rather.

**Comment by KeatonSushi** [comment]
I fixed the navbar.

**Comment by LitterPic** [comment]
I dont see it fixed yet.  We can connect tomorrow if you like.

**Comment by KeatonSushi** [comment]
Okay & I updated them in the aboutusNew.html & donateNew.html so I wouldn't
interfere with the regular files you already had done.

**Comment by KeatonSushi** [comment]
I should be home around 4 or 5pm

**Pull request closed without merging**